client/src/components/Logo.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 
-const Logo = () => {
-  return <div className="logo">/LOGO/</div>;
+const Logo = ({ size }) => {
+  return (
+    <div className="logo">
+      <img
+        src="http://localhost:8080/asset/uni-logo.png"
```

Comment:
jak w poprzednim PRze localhost
Poza tym jak masz to samo logo, którego używasz w wielu miejscach to może warto mieć też to wydzielone - wtedy przy zmianie logo wystarczy jedną lokalizację zmienić